x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/prebuilt_rule_customization/customization_enabled/import_rules.ts

```diff
@@ -159,27 +309,43 @@ export default ({ getService }: FtrProviderContext): void => {
               rule_source: { type: 'internal' },
               immutable: false,
             }),
-            expect.objectContaining({
-              rule_id: prebuiltRuleIds[0],
-              version: 1234,
-              rule_source: { type: 'external', is_customized: true },
-              immutable: true,
-            }),
             expect.objectContaining({
               rule_id: 'custom-rule-2',
               version: 1,
               rule_source: { type: 'internal' },
               immutable: false,
             }),
             expect.objectContaining({
-              rule_id: prebuiltRules[3].rule_id,
-              version: prebuiltRules[3].version,
+              rule_id: 'rule-1',
+              version: 2,
+              rule_source: { type: 'external', is_customized: true },
+              immutable: true,
+            }),
+            expect.objectContaining({
+              rule_id: 'rule-2',
+              version: 2,
               rule_source: { type: 'external', is_customized: false },
               immutable: true,
             }),
           ])
         );
       });
+
+      it('imports prebuilt rules when the rules package is not installed', async () => {
+        await deletePrebuiltRulesFleetPackage({ supertest, es, log, retryService }); // First we delete the rule package
```

Comment:
I'm not sure if this is the correct test setup.

- We don't install the package during the test setup.
- If the package is installed, deleting it causes Fleet to delete all the rule asset saved objects. I don't know if this happens if the package is not actually installed.
- We don't make sure to reinstall the assets. We do it only once in the top-level `before` block for the whole file. This breaks isolation between this test and the rest of the tests in this file.

Please skip the test, I'll try to reimplement it.